### DISCUSSION TITLE
Regarding PyOxidizer

### DISCUSSION THREAD

**Discussion by mert-kurttutan** [Marekkon5/onetagger#348] [General]
Hi,
Thanks for your project.
I am looking for tool to wrap python app into an executable. I also use Rust frequently.
Your project seems to be one of the projects that is active and uses PyOxidizer.
But it looks like PyOxidizer is not maintained in last year.
I want to know about your opinion on this case.

**Comment by Marekkon5** [comment]
Hello, 1T uses Py(thon/Oxidizer) in a different way; we embed Python interpreter into 1T. If you wanna wrap Python app into an executable you should consider PyInstaller or simillar.

**Comment by mert-kurttutan** [comment]
Why do you suggest PyInstaller ? Is it because it is not maintained recently?

From the links below, it seems that PyOxidizer can also be used for distributing python applications as binaries. 

https://pyoxidizer.readthedocs.io/en/stable/pyoxidizer_overview.html
https://pyoxidizer.readthedocs.io/en/stable/pyoxidizer_comparisons.html

**Reply by Marekkon5** [reply]
I suggested PyInstaller because:

1. I've seen it used way more in the wild
2. It has more support than PyOxidizer
3. PyOxidizer is less maintained
4. I've used PyInstaller before, but not PyOxidizer (to dist Python binaries).

**Reply by mert-kurttutan** [reply]
Great, thanks for the clearification

**Comment by Marekkon5** [comment]
I'll close this for now, since unrelated. Feel free to comment or open new issues if you have more questions.
